colossalai/inference/kv_cache/kvcache_manager.py

```diff
@@ -269,7 +273,7 @@ def _allocate_on_block(self, block: CacheBlock, space_asked: int) -> int:
         Returns:
             The remaining space required to be allocated (in other blocks).
         """
-        assert block.available_space > 0, "No available space on block to allocate."
+        assert block.available_space > 0, "No available space on block to allocate but requries allocation."
```

Comment:
What does this mean?

Reply:
it means that block has no available space but we still requires allocation in a step, it's a bug

Reply:
You can phrase like 

```text
Tried to allocate some space but found that there is no available space in blocks.
```

Reply:
fixed